src/components/Checkout/OrderTotals.vue

```diff
@@ -53,6 +53,12 @@
 				>
 					Apply
 				</button>
+				<div v-if="hasUPCCode" class="upc-disclaimer">
```

Comment:
I'm guessing `hasUPCCode` is set somewhere already?

It's a very minor nitpick, but the C in UPC is already "Code", so this is kind of like `hasATMMachine` :P Not going to block the PR by any means, though!

Reply:
yea kinda silly to repeat...